README.md

```diff
@@ -24,8 +24,8 @@
 structures designed to make working with "relational" or "labeled" data both
 easy and intuitive. It aims to be the fundamental high-level building block for
 doing practical, **real world** data analysis in Python. Additionally, it has
-the broader goal of becoming **the most powerful and flexible open source data
-analysis / manipulation tool available in any language**. It is already well on
+the broader goal of becoming **the most powerful and flexible open-source data
+analysis/manipulation tool available in any language**. It is already well on
```

Comment:
These 'corrections' are subjective. I prefer 'open source' over 'open-source' and I prefer 'a / b' rather than 'a/b'.
In the absence of definitive corrections I always prefer to leave as is rather than add a commit